### ISSUE TITLE
Quest: Mekanism - Part 1 > "Becoming Better"

### ISSUE THREAD

**Issue by Mikitiril** [NillerMedDild/Enigmatica2#544]
**Modpack version**
1.70

**Can the issue be reproduced?**
In Single Player - Yes
In Multiplayer - not tested - Edit. Yes, it's bugged in multiplayer too.

**Issue**
Quest: Mekanism - Part 1 > "Becoming Better"
Task : 
- 16x 'Advanced Universal Cable'
- 1x 'Advanced Energy Cube'

Does not detect 'Advanced Energy Cube'. 

Already tried:
- cheating all 'Energy Cubes' both empty and full. 
- upgrading 'Basic Energy Cube' to Advanced with 'Advanced Tier Installer'
- picking up dropped 'Advanced Energy Cube'
- crafting 'Advanced Energy Cube'
- autocrafting 'Advanced Energy Cube'
- editing NBT tags.

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by Xenox40** [comment]
It might be caused by NBT tags. I could only craft/cheat a 1 or 3 NBT tags version, but the one in quest book has 2 NBT tags.

**Comment by kirigiiri** [comment]
Heyo! I was going to write up a new issue but since this sort of covers part of what I ran into, I'll just add on here. I'm experiencing the exact same issue with the exact same quest under Mekanism Part 1 "Becoming Better" and also the quest "Smashing The Power!" which funnily enough is right above the aforementioned one. I tried everything that the person who opened the issue did with both sets of quests - it seems to be a bit buggy in that quest area for whatever reason.
